### PR TITLE
Added "first boot" explanation

### PR DESCRIPTION
**Pull request by kayrus**
resolves https://github.com/coreos/bugs/issues/932

P.S. Just for the history: 
* https://github.com/coreos/bootengine/blob/master/dracut/30ignition/ignition-generator
* https://github.com/coreos/scripts/blob/master/build_library/grub.cfg

/cc @crawford @joshix @robszumski

### CODE REVIEW

File: doc/getting-started.md [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
 
-[config spec]: configuration.md
+## How Ignition is Being Executed ##
+
+One of the main difference between Ignition and [Cloud-Config] is Ignition will run only once at the very first boot. GRUB checks the disks for the dummy disk UUID and sets `coreos.first_boot=1` parameter accordingly. This parameter is being processed by [systemd-generators] every boot inside [initramfs] and whether this parementer is set Systemd adds `ignition.target` as one of the `initrd.target` requirement to run Ignition. In case when you use [PXE][supported platforms] deployment `coreos.first_boot=1` parameter needs to be set manually otherwise Ignition won't run.
```
Comment (SpencerBrown) [inline doc/getting-started.md]: s/whether/whenever/

Comment (kayrus) [inline doc/getting-started.md]: Thanks. Fixed.

**Comment by crawford** [comment]
This is not the appropriate repo for these docs. This PR is related to system integration, so it belongs in coreos/docs/ignition. I'm going to close this and let you reopen another PR over there. Thanks for taking a stab at this.

**Comment by kayrus** [comment]
BTW: I was mixed up between these two repos.

**Pull request closed without merging**